web_poet/rules.py

```diff
@@ -17,7 +17,7 @@
 
 Strings = Union[str, Iterable[str]]
 
-PageObjectRegistryTV = TypeVar("PageObjectRegistryTV", bound="PageObjectRegistry")
+RulesRegistryTV = TypeVar("RulesRegistryTV", bound="RulesRegistry")
```

Comment:
For completeness of backwards-compatibility, we could also have `PageObjectRegistryTV` assuming users were relying on it.

Reply:
That's a good point. I'm not sure how to deprecate PageObjectRegistryTV though. It's also not intended to be used, and I doubt anyone is using it at the moment, given that it's early days.

Reply:
> I'm not sure how to deprecate PageObjectRegistryTV though.

Me neither. But I am thinking we could consider its deprecation implied by the deprecation of all API members that use it, and remove them along those API members.

I also thought about making these variables private, but seeing how scrapy-poet subclasses, it seems like it makes sense to keep them public.